appveyor.yml

```diff
@@ -26,7 +26,7 @@ build_script:
 - stack setup --no-terminal > nul
 # Appveyor has a timeout of 60 mins, building can take longer, limit the time and make sure this succeeds,
 # previous work will get cached and finish on next commit
-- bash -lc "timeout 2700 'C:\projects\postgrest\stack.exe' build -j1 --copy-bins --local-bin-path . || true"
+- bash -lc "timeout 2700 'C:\projects\postgrest\stack.exe' build -j1 --copy-bins --local-bin-path . || (($?==124))"
```

Comment:
Ah, so timeout exits with 124: https://linux.die.net/man/1/timeout

Didn't know that was possible. Great that you have it fixed.